### ISSUE TITLE
Appending from a previous thread

### ISSUE THREAD

**Issue by crazymedguy** [SkepticMystic/breadcrumbs#325]
So while working, I realised a new note I was working could be continued from a previous note. I'm just wondering if there's a way to automatically generate next-prev by selecting a pre-existing page, so that "next::" shows up in this other pre-existing note and "prev::" shows up on the current page. Thank you for the great work! :)

**Comment by SkepticMystic** [comment]
Yeah, this is interesting! It's kind of a combination of the Threading and Freeze commands. 
If you haven't seen, I've rewritten Breadcrumbs from scratch, more info here: https://github.com/SkepticMystic/breadcrumbs/tree/master?tab=readme-ov-file#list-notes
I've implemented _most_ features already, but some I'm still working on, and Threading is one. I'll probably add it back, and this can be one of the options :)
